src/module/Normalizer/src/Normalizer/Adapter/EntityAdapter.php

```diff
@@ -181,8 +181,8 @@ protected function getHeadTitle()
             $parent = $this->getObject()->getParents('link')->first();
             $parentAdapter = new EntityAdapter();
             $parentAdapter->setTranslator($this->translator);
-            $normalizedParent = $parentAdapter->normalize($parent);
-            $parentTitle = $normalizedParent->getTitle();
+            $parentAdapter->normalize($parent);
+            $parentTitle = $parentAdapter->getTitle();
```

Comment:
Because I run into an issue when doing the same thing for the meta description (meta description is done by the adapter, not by the entity). I refactored it here so we are doing the same thing for titles and meta stuff.

Reply:
wouldnt `$normalized->getMetadata()->getMetadescription()` work there as well?